server/calendar/calendar-service.js

```diff
@@ -0,0 +1,59 @@
+var gcal = require('google-calendar');
+
+var calendar = {
+  addCalendarGig: function (user, gigTitle) {
+    var accessToken = user.accessToken;
+    var calendar_id = user.calendar_id;
+    // Renaming catering gigs'events' to conform with Googles API which uses 'events'
+    var eventDescription = gigName;
+    return gcal(accessToken).events.quickAdd(calendar_id, eventDescription).then(function (data) {
+      return data;
+    }).catch(function (err) {
+      console.log('add Calendar Gig error ', err);
+      return {};
+    });
+  },
+
+  removeCalendarGig: function (user, gigId) {
+    var accessToken = user.accessToken;
+    var calendarId = user.calendar_id;
+    // Renaming catering gigs'events' to conform with Googles API which uses 'events'
+    var eventId = gigId;
+    return gcal(accessToken).events.delete(calendar_id, eventId).then( function (data) {
```

Comment:
calender_id will be undefined here, since on line 19 the variable is named calenderId